.gitignore

```diff
@@ -6,6 +6,9 @@ build.properties
 build/**                                                                                                                                                                         
 dist/*.tar.gz
 
+# generated code
+bin/cqlsh
```

Comment:
this commit includes a handful changes. i could guess. but would be ideal this change could be 

- split into smaller commits, each of which is
- accompanied with commit message explaining why we need them and probably how they help with packaging this python module as a wheel.

Reply:
change the commit message, with more details.